### DISCUSSION TITLE
Policies on reviewing and merging

### DISCUSSION THREAD

**Discussion by diazona** [pytest-dev/pytest-localserver#23] [General]
@redtoad @coordt Since I know we're not all going to be active contributors all the time here, I wanted to get a read from you on what should be required before a PR gets merged. Should each of us feel free to merge pull requests as we see fit? Or should we try to get agreement from two or all three of us before making changes? And/or should we have a rule that for PRs made by one of us, someone other than the original author needs to approve it first?

I'm happy to go ahead with whatever works, I just don't want to subvert any expectations you may have.

Once we decide, maybe this could get formalized in a `CONTRIBUTING.md` file or something. I'd be happy to write it up.

**Comment by redtoad** [comment]
I'd say "one of the maintainers" needs to approve

**Comment by coordt** [comment]
That works for me.

**Comment by diazona** [comment]
Thanks for chiming in. Is that one of the maintainers other than the author, or do you mean we should just merge our own PRs (presumably after waiting a reasonable time for comments)?

I'm guessing you meant the former, just want to be clear.

**Reply by coordt** [reply]
I am fine with either or both. I would prefer having another person look it over, but there should be a time limit, after which the author can merge in the branch.

**Reply by diazona** [reply]
Makes sense. Maybe a week? Two weeks? :shrug:

**Reply by diazona** [reply]
@coordt @redtoad any input here?
